src/app/sign-in-module/containers/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,188 @@
+  font-size: 40px;
+}
+.login-label{
+  font-family: $font-family_1;
+  font-size: 20px;
+  font-weight: 600;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  text-align: left;
+}
+.login-input {
+  height: 40px;
+  border-style: solid;
+  border-color: black !important;
+  border-radius: 5px;
+  margin: 5px;
+  padding: 5px;
+}
+.login-button-action {
+  margin: 5px 40px;
+  padding: 10px;
+
+}
+
+.login-err {
+  color: red;
+}
+.login-content {
+  width: full;
+  height: 700px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #24c6be;
+}
+.login-button-rondom {
+  margin: 5px;
+  padding: 10px;
+}
+#logincard {
+  background: #ffffff;
+  border-radius: 8px;
+  box-shadow: 1px 2px 8px rgba(0, 0, 0, 0.65);
+  height: 400px;
+  filter: brightness(1.3);
+}
+
+@media (min-width: 450px) {
+  #logincard {
+    width: 500px;
+  }
+}
+@media (min-width: 350px) {
+  #logincard {
+    width: 500px;
+  }
+}
+@media (min-width: 800px) {
+  #logincard-content {
+    padding: 12px 44px;
+  }
+}
+@media (min-width: 600px) {
+  #logincard-content {
+    padding: 12px 24px;
+  }
+}
+#logincard-title {
+  font-family: $font-family_2;
+  letter-spacing: 4px;
+  padding-bottom: 5px;
+  padding-top: 5px;
+  text-align: center;
+}
+
+@media (min-width: 600px) {
+  #logincard-title {
+    padding-top: 15px;
+  }
+}
+#logincard-title h2 {
+  font-size: 40px;
+  font-weight: 500;
+}
+
+#submit-btn {
+  background: -webkit-linear-gradient(right, #7be9f7, #2d67bd);
+  border: none;
+  border-radius: 21px;
+  box-shadow: 0px 1px 8px #24c6be;
+  cursor: pointer;
+  color: $color_2;
+  font-family: $font-family_3;
+  height: 42.3px;
+  margin: 0 auto;
+  margin-top: 30px;
+  transition: 0.25s;
+  width: 153px;
+  &:hover {
+    box-shadow: 0px 1px 18px #249bc6;
+  }
+}
+
+.form {
+  align-items: left;
+  display: flex;
+  flex-direction: column;
+}
+.usernamelable {
+  font-family: $font-family_1;
+  font-size: 20px;
+  font-weight: 600;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  text-align: left;
+}
+.form-border {
+  background: -webkit-linear-gradient(right, #7be9f7, #2d67bd);
+  height: 1px;
+  width: 100%;
+}
+.form-content {
+  background: #fbfbfb;
+  border: solid rgb(151, 150, 150);
+  height: 40px;
+  font-size: large;
+  border-radius: 21px;
+}
+
+.grid-container {
+  display: grid;
+  gap: 2rem;
+  padding: 10px;
+}
+@media (min-width: 600px) {
+  .grid-container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+@media (min-width: 800px) {
+  .grid-container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+.grid-item {
+  gap: 5rem;
+}
+
+#random-button {
+  background: -webkit-linear-gradient(right, #7be9f7, #2d67bd);
+  border: none;
+  border-radius: 21px;
+  box-shadow: 0px 1px 8px #24c6be;
+  cursor: pointer;
+  color: $color_2;
+  font-family: $font-family_3;
+  height: 42.3px;
+  transition: 0.25s;
+  width: 153px;
+  &:hover {
+    box-shadow: 0px 1px 18px #249bc6;
+  }
+}
```

Comment:
make whole file indentation consistence. in some space there is space between classes. Some places don't have any space. Your code reflects you.

Reply:
done